authorization.spec.ts

```diff
@@ -18,23 +18,29 @@ pax2payClient && (pax2payClient.organization = "agpiPo0v")
 describe("pax2pay.Authorization", () => {
 	let authorization: pax2pay.Authorization | undefined
 	let failedAuthorization: pax2pay.Authorization | undefined
+	const now = isoly.DateTime.now()
 	beforeAll(async () => {
-		const yesterday = isoly.DateTime.previous(isoly.DateTime.now(), { hours: 1 })
+		const yesterday = isoly.DateTime.previous(now, { hours: 1 })
```

Comment:
Not really yesterday.

Reply:
yesterday is your fault, for that im making it yesterhour

Reply:
omg.